app/controllers/RequiredGiinController.scala

```diff
@@ -43,7 +44,7 @@ class RequiredGiinController @Inject() (
     extends FrontendBaseController
     with I18nSupport {
 
-  val form = formProvider()
+  val form: Form[String] = formProvider()
```

Comment:
here

Reply:
also resolving a warning just to add type